src/components/TeamMemberList.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import TeamMember from "./TeamMember";
+import axios from "axios";
+
+const TeamMemberList = (props) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
+      setUsers(response.data);
+    };
+    fetchUsers();
+  },[]);
+
+  return (
+    <>
+      {users.map((user) => {
```

Comment:
what would happen if fetch fail? nothing, user won't know that something went wrong
solution is handle catch in fetching and setting some local state for error
`isError ? render some info "fetching went wrong, sorry refresh page" : null`

Reply:
Changed TeamMemberList component to support error handling - there will be a <p>Error connecting to user database, please contact site administrator.</p> message if data is not fetched.